ptmdmusique/cs464-564-final-project
# Added more charts to the Statistics page

## Describe your changes

- Added 2 new doughnut charts (Color and Habitat)
- Added 3 new bar charts (HP, Attack, and Defense)

## Step to test
- Make sure all new charts show up on the side menu
- Make sure all new chart render properly
- Make sure the toggle bar works for all new bar charts
- Make sure the PokemonTable is rendered when you click on a cell for all new doughnut charts

## Issue ticket number and link



- Completes issue #4 

## Checklist before requesting a review
